website/blog/2022-12-01-release-0.10-blog.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Podman Desktop Release v0.10
+description: Podman Desktop v0.10 has been released!
+slug: podman-desktop-release-0.10
+authors: [deekay2310]
+tags: [podman-desktop, release, kubernetes]
+hide_table_of_contents: false
+---
+
+This blog post covers all that is new with the latest release of Podman Desktop v0.10!
```

Comment:
I would say it's not needed to say it's a blog post as we're in the blog section.

Reply:
https://github.com/containers/podman-desktop/pull/943/commits/9ae20742cadf241c250c5faff504b3738d335f16